packages/flutter_app/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: |
   The Flutter App Template is a starter kit for developing Flutter apps.
   It is also a project that Altive Inc continues to update as a sample of architecture, configuration, and package usage.
 publish_to: "none"
-version: 1.0.0+8
+version: 1.0.0+29
```

Comment:
Since the latest build number uploaded to App Store Connect was 28, I set the next one to 29!